### PR TITLE
remove helper classes

### PR DESCRIPTION
**Pull request by jakerichan**
@jaydenanderson @momotofu

### CODE REVIEW

File: components/toolbar/index.ess [hunk 1]
```diff
@@ -1,4 +1,7 @@
 .Toolbar
+  display flex
+  align-items stretch
+  flex-direction column
```
Comment (mndvns) [inline components/toolbar/index.ess]: *shakes head in disappointment*

File: components/toolbar/index.ess [hunk 2]
```diff
@@ -121,11 +136,3 @@ hr.Toolbar-block-center
...
       color = __.color.primary
       transition none
-
-.lte-sm-hide
-  @media all and (max-width: 599px)
-    display none
-
-.gt-sm-hide
-  @media all and (min-width: 600px)
-    display none
```
Comment (mndvns) [inline components/toolbar/index.ess]: :+1:

**Comment by mndvns** [comment]
i would double check that you're actually using this... if i remember right, this is not actually used by any app anywhere. 

i think we realized that the maker-ui was the only app that needed or wanted this, and we moved the component there.

**Comment by mndvns** [comment]
i may be thinking of the Side component

**Comment by jakerichan** [comment]
Yeah, we use this one.  We actually have one in movement-ui-kit we should remove.

**Pull request merged**